src/Microsoft.TemplateEngine.Abstractions/ILocalizationModel.cs

```diff
@@ -34,8 +34,8 @@ public interface ILocalizationModel
 
         /// <summary>
         /// Gets the localization models for the post actions defined in this template.
-        /// The keys represend the indices of the post actions as they appear in the template config file.
+        /// The keys represent the id of the post actions.
         /// </summary>
-        IReadOnlyDictionary<int, IPostActionLocalizationModel> PostActions { get; }
+        IReadOnlyDictionary<string, IPostActionLocalizationModel> PostActions { get; }
```

Comment:
1) this is a public API change that we are trying to avoid
2) `ILocalizationModel` is interface internal to RunnableProjects. Should we move it (and `IPostActionLocalizationModel`) to `RunnableProjects` and make internal?

Reply:
If it is internal to Runnable projects, sounds like a good idea. I will do it.